yarn.lock

```diff
@@ -10239,6 +10269,15 @@ react-is@^18.0.0, react-is@^18.2.0:
   resolved "https://registry.yarnpkg.com/react-is/-/react-is-18.2.0.tgz#199431eeaaa2e09f86427efbb4f1473edb47609b"
   integrity sha512-xWGDIW6x921xtzPkhiULtthJHoJvBbF3q26fzloPCK0hsvxtPVelvftw3zjbHWSkR2km9Z+4uxbDDK/6Zw9B8w==
 
+react-native-xaml@^0.0.70:
```

Comment:
Lets make sure we are bringing in the same versions of things as the other packages. - We probably shouldn't need any new entries in the lock file if everything aligns.